tests/scripts/common_functions.sh

```diff
@@ -174,8 +177,8 @@ function deploy_autotune() {
 	${cmd}
 	
 	status="$?"
-	# Check if autotune is deployed 
-	if [ "${status}" -eq "1" ]; then
+	# Check if autotune is deployed(Ignore in case of configmap_yaml_tests)
```

Comment:
Include a space here before braces and elaborate why it needs to be ignored for config map yaml tests